content/docs/user-guide/packaging/project-export-cli.md

```diff
@@ -0,0 +1,254 @@
+---
+linkTitle: Project Export CLI
+title: Project Export CLI Tool
+description: Learn how to use the Project Export CLI to automate preparing your project for release.
+toc: true
+weight: 400
+---
+With the new `export-project` CLI command, it is now possible to automate the packaging and release of game projects.
+
+{{< note >}}
+If you wish to learn how to prepare projects manually for Windows, please consult the page [Creating a Project Game Release Layout for Windows.](/docs/user-guide/packaging/windows-release-builds)
+{{< /note >}}
+
+## Prerequisites
+To make best use of the `export-project` command, it is recommended to have a project with at least one starting scene, and all necessary seedlist files prepared. AssetProcessor Registry file settings may need to be tweaked. To learn how to set this up, please consult the following page: [Creating a Project Game Release Layout for Windows, section: Set the Starting Level](windows-release-builds/#set-the-starting-level).
```

Comment:
Suggest hyperlinking “seedlist” to seedlist documentation

Reply:
we couldn't find exact documentation for seedlist, but seed files were mentioned as part of the asset bundler overview, so we will link to that.